classes/imx-boot-container.bbclass

```diff
@@ -28,6 +28,16 @@
 ATF_MACHINE_NAME = "bl31-${ATF_PLATFORM}.bin"
 ATF_MACHINE_NAME:append = "${@bb.utils.contains('MACHINE_FEATURES', 'optee', '-optee', '', d)}"
 
+IMX_BOOT_CONTAINER_FIRMWARE_SOC = ""
+IMX_BOOT_CONTAINER_FIRMWARE_SOC:mx8mq-generic-bsp = " \
+    signed_dp_imx8m.bin \
+    signed_hdmi_imx8m.bin \
```

Comment:
Who is the provider of this firmware? It shall participate in `IMX_EXTRA_FIRMWARE`, so that the dependencies are resolved in `do_resolve_and_populate_binaries[depends]` invocation.

Otherwise, the build might break as `do_resolve_and_populate_binaries ()` would not be able to find those blobs in some cases.

Reply:
It's already taken care of as part of firmware-imx:
https://github.com/Freescale/meta-freescale/blob/master/recipes-bsp/firmware-imx/firmware-imx-8m_8.17.bb#L17

Reply:
Great, thanks for confirming this! 👍🏻